addons/account_tax_python/account_tax_python.py

```diff
@@ -25,7 +27,7 @@ class AccountTaxPython(models.Model):
     def _compute_amount(self, base_amount, price_unit, quantity=1.0, product=None, partner=None):
         self.ensure_one()
         if self.amount_type == 'code':
-            localdict = {'base_amount': base_amount, 'price_unit':price_unit, 'quantity': quantity, 'product':product, 'partner':partner}
+            localdict = {'base_amount': base_amount, 'price_unit':price_unit, 'quantity': quantity, 'product':product, 'partner':partner, 'company': company}
```

Comment:
Oups, just took a commit we did very quickly with Fabien without rereading it.

The company is needed because in some countries (in our case Colombia) , the tax amount depends on propoerties of the company.